src/Mod/TechDraw/Gui/mrichtextedit.cpp

```diff
@@ -1,5 +1,5 @@
 /*
-** Copyright (C) 2013 Jiří Procházka (Hobrasoft)
+** Copyright (C) 2013 Jiri Prochazka (Hobrasoft)
```

Comment:
Please keep the encoding of the file as UTF-8

Reply:
Should be ok now, would you please check?